### ISSUE TITLE
Sidebar - Values 3 - Why does Mozilla care about Internet privacy?

### ISSUE THREAD

**Issue by FrancescoSTL** [mozilla/all-aboard#21]
Sidebar 3 for values track

## Flow

User clicks on add-on icon
Side bar is opened containing content
User receives the relevant token
User selects a radio button option from the multiple choice question
User selects "Tell me"
The correct answer is revealed

**Comment by schalkneethling** [comment]
@chrismore In the copy doc for this sidebar, it is stated that we should follow up showing the answer with private browsing. Now, my question is:

It also states that the button copy for this sidebar is Learn more and not Tell me, like the others. I am wondering, should we have the initial copy state Tell me, and once clicked change it to Learn more. When the user then clicks on the button again, it triggers the UITour visual indicating where to open a new private browsing.

Other than that, the other would be to show additional copy when the user clicks the button, which then also shows a secondary button that triggers the UITour. Let me know, thanks!

**Comment by chrismore** [comment]
@schalkneethling good question. I changed the copy to "Tell Me" for the button CTA to align it with the rest of the questions. 

As for the secondary copy, 3 sidebars have follow up copy that would only show after answering the question. I updated the copy that should show what copy and link will show below the button after you answer the question:

https://docs.google.com/document/d/1Fzxe-Y2ZOheJ0vFFjcQxAiofOWMWgBfjWgeG4gTtP-g/edit#

The link in the follow up copy will open the Fx menu and highlight PB.
